en/docs/assets/versions.json

```diff
@@ -11,7 +11,7 @@
         "notes": "7.1.0/micro-integrator/overview/about-this-release-7.1.0"
     },
     "7.0.0": {
-        "doc": "7.0.0",
+        "doc": " https://ei.docs.wso2.com/en/7.0.0/",
         "notes": "7.0.0/micro-integrator/overview/about-this-release-7.0.0"
```

Comment:
We have to do the same thing for the about the release link too

Reply:
Fixed.